spec/features/tslr_claim_spec.rb

```diff
@@ -159,4 +159,120 @@
     visit claim_path("qts-year")
     expect(page).to have_current_path(root_path)
   end
+
+  context "editing a response" do
+    let(:claim_school) { create(:school, :tslr_eligible, name: "Claim School") }
+    let(:current_school) { create(:school, :tslr_eligible, name: "Current School") }
+
+    let(:claim) do
+      claim = create(:tslr_claim,
+        :eligible_and_submittable,
+        employment_status: :different_school,
+        claim_school: claim_school)
+      claim.current_school = current_school
+      claim.save
+      claim
+    end
+
+    before do
+      allow_any_instance_of(ClaimsController).to receive(:current_claim) { claim }
+      visit claim_path("check-your-answers")
+    end
+
+    scenario "Teacher can edit a field" do
+      old_number = claim.national_insurance_number
+      new_number = "AB123456C"
+
+      expect {
+        find("a[href='#{claim_path("national-insurance-number")}']").click
+        fill_in "National Insurance number", with: new_number
+        click_on "Continue"
+      }.to change {
+        claim.reload.national_insurance_number
+      }.from(old_number).to(new_number)
```

Comment:
Nice! Didn't know you could do this.

Reply:
Yeah, it's pretty cool. I've never used it in features before, but seems to work 😄